package-lock.json

```diff
@@ -10,43 +10,20 @@
       "license": "Apache-2.0",
       "dependencies": {
         "@iarna/toml": "^2.2.5",
-        "@rollup/plugin-alias": "^3.1.2",
```

Comment:
I feel like I've done something wrong with this file? Might be lerna releated, never used it lerna before :dagger: